### DISCUSSION TITLE
Hide some plugins required by @theia/plugin-ext

### DISCUSSION THREAD

**Discussion by lstkz** [eclipse-theia/theia#8873] [General] [answered]
@theia/plugin-ext-vscode requires following plugins:
- @theia/debug
- @theia/callhierarchy
- @theia/scm

![image](https://user-images.githubusercontent.com/4478684/102371401-71c4ef80-3fbe-11eb-9472-5dad14ac1bc5.png)

I am trying to simplify the IDE and hide the selected icons. How to do it?

**Comment by vince-fugnitto** [comment, marked as answer]
@lstkz the following discussion and comment may provide you with the information you are searching for.
If you are looking into removing `views` from your application, you can do the following in a custom `@theia` extension:
- https://github.com/eclipse-theia/theia/discussions/8293#discussioncomment-50435

**Reply by lstkz** [reply]
Thanks a lot! "Views" was the right keyword to search for.
